tutorials/tour/abstract-types.md

```diff
@@ -5,11 +5,10 @@ title: Abstract Types
 disqus: true
 
 tutorial: scala-tour
-num: 2
-outof: 35
```

Comment:
Believe it or not this is actually somehow a little robust. It at least means no broken links...